lib/ooxml_parser/xlsx_parser/xlsx_data/view_model/workbook.rb

```diff
@@ -24,11 +25,14 @@ class XLSXWorkbook < CommonDocumentStructure
     attr_accessor :pivot_caches
     # @return [Array<PivotTableDefintion>] list of pivot table defitions
     attr_accessor :pivot_table_definitions
+    # @return [Array<DefinedName>] list of defined names
+    attr_accessor :defined_names
```

Comment:
User `attr_reader` if this possible

This is more safe way to add attributes if no need to change them